clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoOptions.java

```diff
@@ -100,12 +100,12 @@ public Options options() {
     options.addOption(createArgOption(AUTO, "column value auto-increments (true/false)"));
     options.addOption(createArgOption(DEFAULT, "default column value"));
     options.addOption(createSimpleOption("o", OWNER, "display entity owner"));
-    options.addOption(createArgsOption("r", ROLE, "role name"));
     options.addOption(createArgOption(COLUMNFILE, "CSV file describing columns"));
 
     // Properties and tags can have multiple values
```

Comment:
You might want to update this to "Options that support multiple values" or similar